app/lib/utils/debuggingTools

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Would it be cleaner to simply extend the debug library with these methods instead of creating a new module?